### PR TITLE
Two save buttons

### PR DESCRIPTION
**Pull request by vinhtruong94**

### CODE REVIEW

File: app/Http/Requests/Setting/UpdateSettingRequest.php [hunk 1]
```diff
@@ -18,15 +18,13 @@ class UpdateSettingRequest extends Request
...
-        // done in the middleware
         return true;
     }
 
     public function rules()
     {
-        //workaround to conver numeric value to int
-        if (is_numeric($this['value'])) {
+        //workaround for beta message value.
+        if (($this['value']) === '1' || ($this['value']) === '0') {
```
Comment (psaunders) [inline app/Http/Requests/Setting/UpdateSettingRequest.php]: Why? What if it's 3?

File: app/Jobs/Setup/SetDefaultSettings.php [hunk 2]
```diff
@@ -78,6 +78,11 @@ public function handle()
             ]);
 
         Setting::ifNotSetThenSet('theme', [
+                'customTheme' => 1,
+                'predefined'  => [
+                    'primary' => '',
```
Comment (psaunders) [inline app/Jobs/Setup/SetDefaultSettings.php]: Might as well set a default

Comment (vinhtruong94) [inline app/Jobs/Setup/SetDefaultSettings.php]: yeah purple green I will set

File: app/Http/Requests/Setting/UpdateSettingRequest.php (hunk 1 above)
Comment (vinhtruong94) [inline app/Http/Requests/Setting/UpdateSettingRequest.php]: It's back to normal now, I was testing and forgot to set it back to normal

**Pull request merged**